src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/markup/Table.java

```diff
@@ -469,19 +430,17 @@ private HtmlTree createTab(String tabId, HtmlStyle style, boolean defaultTab, St
                 .put(HtmlAttr.TABINDEX, defaultTab ? "0" : "-1")
                 .put(HtmlAttr.ONKEYDOWN, "switchTab(event)")
                 .put(HtmlAttr.ID, tabId)
+                .put(HtmlAttr.ONCLICK, "show('" + id + "', '" + (defaultTab ? id : tabId)
```

Comment:
minor style suggestion: place the ONCLICK code nearer the ONKEYDOWN  code, possibly by moving the ID code up in the list.